src/init.cpp

```diff
@@ -1739,7 +1748,9 @@ bool AppInitMain(const CoreContext& context, NodeContext& node, interfaces::Bloc
         }
     }
 
+    assert(!::masternodeSync);
```

Comment:
Implicit conversion to `bool` seems to be the style inherited from Bitcoin, see examples [here](https://github.com/dashpay/dash/blob/10ddf62dfb2c0ae673c182ea840d9ce0a1b3e635/src/init.cpp#L490), [here](https://github.com/dashpay/dash/blob/10ddf62dfb2c0ae673c182ea840d9ce0a1b3e635/src/init.cpp#L1630) and [here](https://github.com/dashpay/dash/blob/10ddf62dfb2c0ae673c182ea840d9ce0a1b3e635/src/init.cpp#L1686-L1704).

We can change it to the explicit style for Dash-specific code but I'd suggest that be a separate commit since those changes would have be permeated to every Dash-based manager (the commit so far only adds one additional assert). Should we proceed with that?

Reply:
ok, nvm